nifi-framework-api/src/main/java/org/apache/nifi/components/AsyncLoadedProcessor.java

```diff
@@ -17,14 +17,18 @@
 
 package org.apache.nifi.components;
 
-public interface AsyncLoadedProcessor {
+import org.apache.nifi.processor.Processor;
+
+public interface AsyncLoadedProcessor extends Processor {
     default boolean isLoaded() {
         return getState() == LoadState.FINISHED_LOADING;
     }
 
     LoadState getState();
 
     enum LoadState {
+        INITIALIZING_PYTHON_ENVIRONMENT,
```

Comment:
Should this be shortened to `INITIALIZING_ENVIRONMENT`?

Reply:
Or perhaps `INITIALIZING_PROCESSOR_ENVIRONMENT`? The other states do not mention Python specifically.